### PR TITLE
Transfer data excel file

### PR DESCRIPTION
**Pull request by HUANG0421**
I've done 20 entries at the Act Files, and it can be combined with Mira's entries.
> Pull-requests should be ready to merge, and quick to review. Please help your reviewer by using this form. If you require a specific reviewer you can select them on the right. 

### Please indicate relevant issues
See # .
Close # .

### Checklist
-   [x] The title line explains what these changes do
-   [x] All my changes are marked in the `last_modfied` column
-   [x] All formatting changes are intentional (dates, coordinates)
-   [x] My changes do not add empty rows
-   [x] I checked the modified table on GitHub (click `view`)

### CODE REVIEW

File: data/ActType.csv [hunk 1]
```diff
@@ -19,4 +19,6 @@ AT19,to discuss,,2018-02-08,OS,2018-02-08,OS
 AT20,to perform,e.g. a play,2018-02-09,OS,2018-02-09,OS
 AT21,to quote,,2018-02-09,OS,2018-02-09,OS
 AT22,to remember ,,2018-02-13,OS,2018-02-13,OS
-AT23,to tell (to sb),,2018-03-28,WH,2018-03-28,WH
+AT23,other peopler to recite,,2018-04-06,WH,2018-04-06,WH
```
Comment (duncdrum) [inline data/ActType.csv]: what does this mean?

File: data/SocialRelation.csv [hunk 2]
```diff
@@ -18,6 +18,3 @@ SR00016,P0291,P0292,S4.2.1.2my ,弟弟,younger brother,SS00248,326,,2018-03-13,M
 SR00017,P0282,P0292,S4.1.2.1f,母亲,mother,SS00248,326,,2018-03-13,MG,2018-03-13,MG
 SR00018,P0281,P0292,S4.1.1m,父亲,father,SS00248,326,,2018-03-13,MG,2018-03-13,MG
 SR00019,P0281,P0293,S3.2-,朋友,friend,SS00248,239,they shared a room in a peasant‘s household,2018-03-13,MG,2018-03-13,MG
-SR00020,P0322,P0320,S4.1.1m,父亲,father,SS00268,,,2018-03-21,MG,2018-03-21,MG
```
Comment (duncdrum) [inline data/SocialRelation.csv]: why these deletions? The relationship still stands no?

File: data/Place.csv [hunk 3]
```diff
@@ -207,9 +207,7 @@ PL0204,Lianshui,33.781331,119.260227,2018-03-13,MG,2018-03-14,MG,
 PL0205,Xiangchuan,28.515646,112.134533,2018-03-13,MG,2018-03-14,MG,
 PL0206,Manchester,53.480759,-2.242631,2018-03-13,MG,2018-03-14,MG,
 PL0207,Kempen,51.364994,6.420531,2018-03-14,MG,2018-03-14,MG,
-PL0227,Yanchuan,36.878117,110.193514,2018-03-21,MG,2018-03-21,MG,
```
Comment (duncdrum) [inline data/Place.csv]: carefull you are overwriting data

**Comment by HUANG0421** [comment]
@duncdrum hello and sorry that I was sick in the last days. I've finished solving the conflicts, and here are the entries from Dimitrov to Dvořák. Please check it again. :-)

File: data/ActType.csv [hunk 4]
```diff
@@ -19,4 +19,4 @@ AT19,to discuss,,2018-02-08,OS,2018-02-08,OS
 AT20,to perform,e.g. a play,2018-02-09,OS,2018-02-09,OS
 AT21,to quote,,2018-02-09,OS,2018-02-09,OS
 AT22,to remember ,,2018-02-13,OS,2018-02-13,OS
-AT23,to listen to,,2018-03-23,MG,2018-03-23,MG
+AT23,to be block from reading,,2018-04-11,WH,2018-04-13,WH
```
Comment (duncdrum) [inline data/ActType.csv]: did you replace mira's entry on purpose? or are all instances of `AT23` to be block**ed** from reading?

File: data/Membership.csv [hunk 5]
```diff
@@ -6,4 +6,3 @@ I0005,P0019,SS00129,611,,2018-01-30,OS,2018-01-30,OS
 I0005,P0134,SS00129,611,,2018-01-30,OS,2018-01-30,OS
 I0006,P0173,SS00006,137,,2018-02-22,MG,2018-02-22,MG
 I0007,P0169,SS00207,ONLINE SOURCE,,2018-02-22,MG,2018-02-22,MG
-I0008,P0322,SS00268,,,2018-03-21,MG,2018-03-21,MG
```
Comment (duncdrum) [inline data/Membership.csv]: is deleting this entry correct?

**Pull request merged**